src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -223,6 +224,8 @@ public bool IsEmpty(DocumentId documentId)
                 }
 
                 await SerializeAsync(serializer, project, result.ProjectId, _owner.NonLocalStateName, result.Others).ConfigureAwait(false);
+
+                AnalyzerABTestLogger.LogProjectDiagnostics(project, result);
```

Comment:
what we log here is nothing to do with project or document level errors.
..
my understanding is that we decided to log number of errors a project/document produces

if FSA is off, we only get per document errors count, if FSA is on, then we get per project errors count.

we do it only once ever at the time vsix is first installed. is my understanding right? @jinujoseph  @kuhlenh otherwise, we need to define what you want again.